src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveInstanceofConditionalsPassTest.php

```diff
@@ -60,7 +60,6 @@ public function testProcessInheritance()
         (new ResolveChildDefinitionsPass())->process($container);
 
         $expected = array(
-            array('foo', array('bar')),
```

Comment:
changing an existing test case is always suspicious,
is this really legitimate? won't it introduce BC breaks?
we need to answer these questions carefully

Reply:
This change came from the fact that if a definition has already a method call (line 125: `if (!$definition->hasMethodCall($call[0])) {`), I do not add it again to avoid call a setter multiple times. Hard to say if we should deduplicate or not method calls...

Reply:
> Hard to say if we should deduplicate or not method calls
we should not:
```
$twig->addExtension($ext1);
$twig->addExtension($ext1);
...
```

Reply:
I removed method filtering. Method order has changed, but seems normal since we now allow calls inheritance and they are added at the end of the process.

Reply:
Test order fixed in `da6d7e4`